doc/dune-files.rst

```diff
@@ -1715,25 +1694,25 @@ Examples:
 .. code:: lisp
 
    (dirs *) ;; include all directories
-   (dirs :standard \ ocaml) ;; include all directories except ocaml
-   (dirs :standard \ test* foo*) ;; exclude all directories that start with test or foo
+   (dirs :standard \ ocaml) ;; include all dirs except ocaml
+   (dirs :standard \ test* foo*) ;; exclude all dirs that start with test or foo
```

Comment:
Perhaps put `test` and `foo` in monospace here?
Maybe even the `dirs` in …exclude all the `dirs` that start…

Reply:
It is in the comment in a file so I will leave it out.